stubs/Pillow/PIL/MicImagePlugin.pyi

```diff
@@ -9,7 +9,10 @@ _OleStream: TypeAlias = Any  # olefile.OleStream
 class MicImageFile(TiffImageFile):
     ole: _OleFileIO
     format: ClassVar[Literal["MIC"]]
+    format_description: ClassVar[str]
     fp: _OleStream
-    frame: int
+    frame: int | None
+    images: list[str]
```

Comment:
It looks to me like this is actually a list of sequences of strings, rather than a list of strings: the `path` variable here looks to be a sequence of strings, and the code does `self.images.append(path)`: https://github.com/python-pillow/Pillow/blob/77b316c8f01bac2a7bf8b37a8010732d172ad6dc/src/PIL/MicImagePlugin.py#L56-L59

Reply:
Indeed. A `list` of str to be more exact: https://github.com/decalage2/olefile/blob/master/olefile/olefile.py
```suggestion
    images: list[list[str]]
```

Reply:
Yup, that looks correct!